### PR TITLE
Week 1 Homework

### PR DESCRIPTION
**Pull request by gulyener**
Sorry about handing in late

### CODE REVIEW

File: homework-submission/src/App.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      {
+        id: 3,
+        description: 'Eat breakfast',
+        deadline: '2017-09-09',
+        done: false,
+      },
+    ],
+  };
+
+  markComplete = id => {
```
Comment (ArcoMul) [inline homework-submission/src/App.js]: Cool that you already implemented a click event!

File: homework-submission/src/components/Todos.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import TodoItem from './TodoItem';
+
+const StaticTodos = ({ description, deadline }) => {
```
Comment (ArcoMul) [inline homework-submission/src/components/Todos.js]: You're using the plural work StaticTodos here while it only renders one todo item, might be confusing

**Comment by gulyener** [comment]
Thanks for the review @ArcoMul. I have added the TodoItem to Static List but I could not remove the markComplete function in StaticTodo because TodoItem uses that function I guess. I'm not sure I did it the right way though, is this still static?

**Comment by ArcoMul** [comment]
Super! 👏

**Pull request closed without merging**